web/src/user_topics.js

```diff
@@ -11,54 +11,67 @@ import * as stream_data from "./stream_data";
 import * as timerender from "./timerender";
 import {get_time_from_date_muted} from "./util";
 
-const muted_topics = new Map();
+const topics_dict = new Map();
```

Comment:
We can maybe use `user_topics` instead of `topics_dict` -- because it doesn't store data about every topic.

Reply:
I will wait for feedback on this name in [CZO](https://chat.zulip.org/#narrow/stream/6-frontend/topic/Unmute.20Topic.20UI.20.2324244.20.26.20.2324243/near/1542446) before changing.